pinot-core/src/test/java/org/apache/pinot/queries/BaseQueriesTest.java

```diff
@@ -197,21 +198,31 @@ protected BrokerResponseNative getBrokerResponseForSqlQuery(String sqlQuery, Pla
     }
     queryOptions.put(Request.QueryOptionKey.GROUP_BY_MODE, Request.SQL);
     queryOptions.put(Request.QueryOptionKey.RESPONSE_FORMAT, Request.SQL);
+    BrokerRequest strippedBrokerRequest = GapfillUtils.stripGapfill(brokerRequest);
```

Comment:
(MAJOR) Query options should be preserved when stripping the gapfill

Reply:
(minor) rename it to `serverBrokerRequest`

Reply:
Fixed